tools/protodoc/protodoc.bzl

```diff
@@ -92,4 +91,5 @@ proto_doc_aspect = aspect(
             cfg = "host",
         ),
     },
+    implementation = _proto_doc_aspect_impl,
```

Comment:
I assume this is just some Buildifier artifact..

Reply:
Yes, not sure why `check_format` is eager to refactor all bazel files on my machine.